### PR TITLE
Follow up on CC-1420: Add OCaml language support docs

### PR DESCRIPTION
**Pull request by andy1li**

### CODE REVIEW

File: docs/challenges/language-support/go.mdx [hunk 1]
```diff
@@ -52,9 +55,10 @@ Let's say you wanted to extract `foo.go` into the `bar` package. You'd move `foo
 package bar
 
 fun foo() {
-    println("Hello world!")
+println("Hello world!")
```
Comment (rohitpaulk) [inline docs/challenges/language-support/go.mdx:58]: @andy1li this diff seems wrong, it removes indentation

File: docs/challenges/language-support/go.mdx [hunk 2]
```diff
@@ -52,9 +55,10 @@ Let's say you wanted to extract `foo.go` into the `bar` package. You'd move `foo
...
 }
-```
+
+````
```
Comment (rohitpaulk) [inline docs/challenges/language-support/go.mdx:61]: @andy1li this seems wrong too, uses 4 `s

**Review by rohitpaulk — CHANGES_REQUESTED** [review]
Notes on weird diffs

File: docs/challenges/language-support/go.mdx (hunk 1 above)
Comment (andy1li) [reply docs/challenges/language-support/go.mdx:58]: Hmm, let me find out why it happened and re-check all the other diffs.

**Comment by andy1li** [comment]
Prettier was the culprit. Turned it off and started over.

**Review by rohitpaulk — APPROVED** [review]

**Pull request merged**